otter/test/models/test_cass_models.py

```diff
@@ -3708,36 +3710,53 @@ def setUp(self):
             self.tenant_id, self.group_id)
         self.dt = datetime(2010, 10, 20, 10, 0, 0)
 
-    def _test_get_servers(self, query_result, exp_result):
-        sequence = SequenceDispatcher([
+    def _test_get_servers(self, only_as_active, query_result, exp_result):
+        sequence = [
             (CQLQueryExecute(
-                query=('SELECT server_blob, last_update FROM servers_cache '
+                query=('SELECT server_blob, server_as_active, last_update '
+                       'FROM servers_cache '
                        'WHERE "tenantId"=:tenantId AND "groupId"=:groupId '
                        'ORDER BY last_update DESC;'),
-                params=self.params, consistency_level=ConsistencyLevel.ONE),
-             lambda i: query_result)])
-        dispatcher = test_dispatcher(sequence)
-        with sequence.consume():
-            self.assertEqual(
-                sync_perform(dispatcher, self.cache.get_servers()),
-                exp_result)
+                params=self.params, consistency_level=ConsistencyLevel.QUORUM),
+             lambda i: query_result)]
+        self.assertEqual(
+            perform_sequence(sequence, self.cache.get_servers(only_as_active),
+                             test_dispatcher(sequence)),
+            exp_result)
 
     def test_get_servers_empty(self):
         """
         `get_servers` returns ([], None) if cache is empty
         """
-        self._test_get_servers([], ([], None))
+        self._test_get_servers(True, [], ([], None))
+        self._test_get_servers(False, [], ([], None))
 
-    def test_get_servers(self):
+    def test_get_servers_all(self):
         """
         `get_servers` fetches all servers that have highest last_fetch
         time
         """
         self._test_get_servers(
-            [{"server_blob": '{"a": "b"}', "last_update": self.dt},
-             {"server_blob": '{"d": "e"}', "last_update": self.dt}],
+            False,
+            [{"server_blob": '{"a": "b"}', "last_update": self.dt,
+              "server_as_active": False},
```

Comment:
Non-blocking - do we want to make one of these True (same with `test_get_servers_diff_last_update`) as a sanity check/regression test to make sure we're not filtering out all True's also?